packages/terra-open-source-scripts/src/terra-cli/release/index.js

```diff
@@ -0,0 +1,11 @@
+const releaseMonoRepoHandler = require('./releaseMonoRepoHandler');
+const releaseRepoHandler = require('./releaseRepoHandler');
+const isMonoRepo = require('../../utils/isMonoRepo');
+
+const release = {
+  command: 'release',
+  describe: 'Tag the repo and release',
```

Comment:
Can we be a little more descriptive?

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/460/commits/6c35f4191a2115b9652c22dc9f42105554d0b08c